### ISSUE TITLE
Incompatible with latest rubocop (currently 0.35)

### ISSUE THREAD

**Issue by jaredbeck** [sds/slim-lint#21]
**rubocop 0.34.1**

```
bundle | ggrep -E '(rubo|slim_lint)'
Using rubocop 0.34.1
Using slim_lint 0.6.1
bundle exec slim-lint app/views
echo $? #=> 0
```

**rubocop 0.35.1**

```
bundle | ggrep -E '(rubo|slim_lint)'
Using rubocop 0.35.1
Using slim_lint 0.6.1
An error occurred while Style/AlignParameters cop was inspecting /Users/jared/redacted/app/views/redacted.slim.slim_lint.tmp20160106-48574-t4fjq4.
To see the complete backtrace run rubocop -d.

1 error occurred:
An error occurred while Style/AlignParameters cop was inspecting /Users/jared/redacted/app/views/redacted.slim.slim_lint.tmp20160106-48574-t4fjq4.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
Mention the following information in the issue report:
0.35.1 (using Parser 2.2.3.0, running on ruby 2.2.3 x86_64-darwin14)
An error occurred while Style/AlignParameters cop was inspecting /Users/jared/redacted/app/views/redacted.slim.slim_lint.tmp20160106-48574-1jekafi.
To see the complete backtrace run rubocop -d.
An error occurred while Style/AlignParameters cop was inspecting /Users/jared/redacted/app/views/redacted.slim.slim_lint.tmp20160106-48574-1jekafi.
To see the complete backtrace run rubocop -d.

2 errors occurred:
An error occurred while Style/AlignParameters cop was inspecting /Users/jared/redacted/app/views/redacted.slim.slim_lint.tmp20160106-48574-1jekafi.
An error occurred while Style/AlignParameters cop was inspecting /Users/jared/redacted/app/views/redacted.slim.slim_lint.tmp20160106-48574-1jekafi.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
Mention the following information in the issue report:
0.35.1 (using Parser 2.2.3.0, running on ruby 2.2.3 x86_64-darwin14)
An error occurred while Style/AlignParameters cop was inspecting /Users/jared/redacted/app/views/redacted.slim.slim_lint.tmp20160106-48574-1hzi2bk.
To see the complete backtrace run rubocop -d.

1 error occurred:
An error occurred while Style/AlignParameters cop was inspecting /Users/jared/redacted/app/views/redacted.slim.slim_lint.tmp20160106-48574-1hzi2bk.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
Mention the following information in the issue report:
0.35.1 (using Parser 2.2.3.0, running on ruby 2.2.3 x86_64-darwin14)
echo $? #=> 0
```

It says "To see the complete backtrace run rubocop -d." but if I did that it would just run rubocop, not slim-lint.  Please let me know how I can help provide more information.

**Comment by deivid-rodriguez** [comment]
Confirming several issues with latest RuboCop. @sds You can run `slim-lint` on https://github.com/deivid-rodriguez/nipanipa to reproduce those.

**Comment by jaredbeck** [comment]
Rubocop moves pretty quickly. They're on 0.36 now. My team has decided to drop slim-lint so we can keep up with rubocop.  I hope we'll be able to use slim-lint again in the future.  Thanks for your work on it.

**Comment by sds** [comment]
Fixed in fce3d84d7123aeb9ca74b1414dad4981fc149354. 

@jaredbeck / @deivid-rodriguez can either of you take that for a spin and see if you see any other unexpected problems? Running against @deivid-rodriguez's nipanipa project results in no lints being reported.

**Comment by jaredbeck** [comment]
Shane, I tried `master` and the issue seems resolved: it seems compatible with rubocop 0.36.

```
bundle | ggrep -E '(rubo|slim_lint)'
Using rubocop 0.36.0
Using slim_lint 0.6.1 from git://github.com/sds/slim-lint.git (at master@26b7ba1)
```

I did see a minor issue with incorrect line numbers, but I doubt it's related to this issue. Thanks for fixing this so quickly. Looking forward to a release, thanks!

**Comment by sds** [comment]
Thanks for trying it out, @jaredbeck.

Your mentioning of line numbers being off is interesting. If you could provide an example I'd be happy to take a look!

**Comment by deivid-rodriguez** [comment]
@sds Working great, thanks for the quick fix!

**Comment by sds** [comment]
I went ahead and released [0.7.0](https://github.com/sds/slim-lint/releases/tag/v0.7.0).
